shared/src/commonMain/kotlin/com/bunbeauty/shared/domain/interactor/cart/GetOldTotalCostUseCase.kt

```diff
@@ -0,0 +1,22 @@
+package com.bunbeauty.shared.domain.interactor.cart
+
+import com.bunbeauty.shared.domain.model.cart.CartProduct
+
+class GetOldTotalCostUseCase {
+    operator fun invoke(cartProductList: List<CartProduct>): Int {
+        val oldTotalCost = cartProductList.sumOf { cartProduct ->
+            val sumOfCostAndAdditions =
+                (cartProduct.product.oldPrice ?: cartProduct.product.newPrice) + getAdditionsPrice(
+                    cartProduct
+                )
+            sumOfCostAndAdditions * cartProduct.count
+        }
+
+        return oldTotalCost
+    }
+
+    private fun getAdditionsPrice(cartProduct: CartProduct) =
```

Comment:
Looks like copy/paste from `GetNewTotalCostUseCase`, let's move to separate usecase